### PR TITLE
QMark AI

### PR DESCRIPTION
**Pull request by DeathlyPlays**
I assumed this file was most appropriate for this command.

### CODE REVIEW

**Comment by Spandamn** [comment]
Don't we have /pick? 
Id use !pick yes, no

**Comment by DeathlyPlays** [comment]
But it's the Lord Helix kek

File: chat-plugins/info.js [hunk 1]
```diff
@@ -1874,6 +1874,15 @@ exports.commands = {
 	},
 	pickrandomhelp: ["/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements."],
 
+	helixfossil: 'm8b',
+	helix: 'm8b',
+	magic8ball: 'm8b',
+	m8b: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		let results = ['Signs point to yes.', 'Yes.', 'Reply hazy, try again.', 'Without a doubt.', 'My sources say no.', 'As I see it, yes.', 'You may rely on it.', 'Concentrate and ask again.', 'Outlook not so good.', 'It is decidedly so.', 'Better not tell you now.', 'Very doubtful.', 'Yes - definitely.', 'It is certain.', 'Cannot predict now.', 'Most likely.', 'Ask again later.', 'My reply is no.', 'Outlook good.', 'Don\'t count on it.'];
+		return this.sendReplyBox(results[Math.floor(20 * Math.random())]);
```
Comment (panpawn) [inline chat-plugins/info.js]: ``20``? I didn't count how long ``results`` is, but I would just do ``results.length`` here; that's more readable.

also, lol this command

Comment (DeathlyPlays) [inline chat-plugins/info.js]: Yeah, definitely "original"

**Comment by DeathlyPlays** [comment]
It was 20 for the record @panpawn

File: chat-plugins/info.js [hunk 2]
```diff
@@ -1874,6 +1874,15 @@ exports.commands = {
 	},
 	pickrandomhelp: ["/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements."],
 
+	helixfossil: 'm8b',
+	helix: 'm8b',
+	magic8ball: 'm8b',
+	m8b: function (target, room, user) {
+		if (!this.runBroadcast()) return;
+		let results = ['Signs point to yes.', 'Yes.', 'Reply hazy, try again.', 'Without a doubt.', 'My sources say no.', 'As I see it, yes.', 'You may rely on it.', 'Concentrate and ask again.', 'Outlook not so good.', 'It is decidedly so.', 'Better not tell you now.', 'Very doubtful.', 'Yes - definitely.', 'It is certain.', 'Cannot predict now.', 'Most likely.', 'Ask again later.', 'My reply is no.', 'Outlook good.', 'Don\'t count on it.'];
+		return this.sendReplyBox(results.length);
```
Comment (HoeenCoder) [inline chat-plugins/info.js]: this always returns 20, it should be
```javascript
return this.sendReplyBox(results[Math.floor(Math.random() * results.length)]);
```

File: chat-plugins/info.js [hunk 3]
```diff
@@ -1874,6 +1874,15 @@ exports.commands = {
 	},
 	pickrandomhelp: ["/pick [option], [option], ... - Randomly selects an item from a list containing 2 or more elements."],
 
+	helixfossil: 'm8b',
+	helix: 'm8b',
+	magic8ball: 'm8b',
+	m8b: function (target, room, user) {
```
Comment (HoeenCoder) [inline chat-plugins/info.js]: `8ball` should probably be an alias (imo at least)

File: chat-plugins/info.js (hunk 2 above)
Comment (DeathlyPlays) [inline chat-plugins/info.js]: Ah haha I did :|

**Comment by sparkychildcharlie** [comment]
do it as ``"8ball": "m8b",``

**Comment by sparkychildcharlie** [comment]
Yeah that's how it works, except I'm almost sure that PS wants single quotes instead of double quotes

**Comment by bumbadadabum** [comment]
I don't see the benefit of this command at all.

**Comment by Zarel** [comment]
I understand why, but no.

Maybe if you make an API for QMark though... :p

**Comment by DeathlyPlays** [comment]
QMark?

**Comment by Zarel** [comment]
http://qmarkai.com/

**Comment by DeathlyPlays** [comment]
Does that even have a random answer pool?

**Comment by sparkychildcharlie** [comment]
My only comment about qmarkai would be the fact it could potentially spew some not so child appropriate content that PS works hard to defend against. (see Tay Tweets and what snen does with the learning bots that use some algorithm to pick up words and recreate phrases)

**Comment by DeathlyPlays** [comment]
True...

**Comment by Zarel** [comment]
I made QMark myself, and I can guarantee it won't give inappropriate answers because it's not a learning bot.

**Comment by Zarel** [comment]
Also, what do you mean by "random answer pool"?

**Comment by DeathlyPlays** [comment]
Like does it have set responses that it will reply with?

**Comment by DeathlyPlays** [comment]
Guess I should close this since you don't want it :I

**Comment by Zarel** [comment]
It answers yes/no questions randomly.

**Comment by DeathlyPlays** [comment]
Ah ok

**Comment by DeathlyPlays** [comment]
What are all of the answer messages?

**Comment by HoeenCoder** [comment]
@DeathlyPlays you cant just add all the answer messages to an array, you should try to involve the API. @sparkychildcharlie has managed to get it working on his bot.

**Comment by DeathlyPlays** [comment]
Thats what I planned to do @HoeenCoder

**Comment by DeathlyPlays** [comment]
But now I have a coding task already e.e I'll get to it later >_>

**Comment by DeathlyPlays** [comment]
This may take awhile lol

**Pull request closed without merging**